pom.xml

```diff
@@ -27,7 +27,7 @@
     <changelist>-SNAPSHOT</changelist>
     <devon4j.version>${revision}${changelist}</devon4j.version>
     <devon4j.flatten.mode>oss</devon4j.flatten.mode>
-    <spring.boot.version>2.1.2.RELEASE</spring.boot.version>
+    <spring.boot.version>2.1.6.RELEASE</spring.boot.version>
```

Comment:
OK. so you also fixed the version update in the same PR.
Fine.

Reply:
I now see you also explained that in the top-level comment of this PR. Excellent.